base/applications/rapps/include/winmain.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
-#include <windef.h>
-#include <wininet.h>
+
+// #include <wininet.h>
```

Comment:
WIP leftover?

Reply:
The PR was also a pretext for suggesting which other headers could be factored out / in which files it makes sense to do that.
And yes, this is some leftover.